crates/provider/src/native.rs

```diff
@@ -139,10 +130,10 @@ impl<T: FileSystem + Send + Sync> Provider for NativeProvider<T> {
 
     async fn spawn_node(
         &mut self,
-        node: Node,
-        files_inject: Vec<FileMap>,
-        keystore: &str,
-        db_snapshot: &str,
+        _node: Node,
+        _files_inject: Vec<FileMap>,
```

Comment:
Looks a better name, wdyt ?

```suggestion
        _injected_files: Vec<FileMap>,
```

Reply:
Doesn't really matter for this PR. Once implementation is done we can address the name based on usability

Reply:
I have originally `files_to_inject` / `files_to_get` but I found very verbose. Now I think those are better names to explain what is the reason behind those `inputs`.